python/tidy/servo_tidy/tidy.py

```diff
@@ -601,11 +602,12 @@ def get_wpt_files(only_changed_files, progress):
 
 def check_wpt_lint_errors(files):
     wpt_working_dir = os.path.abspath(os.path.join(".", "tests", "wpt", "web-platform-tests"))
-    site.addsitedir(wpt_working_dir)
-    from tools.lint import lint
-    returncode = lint.lint(files)
-    if returncode:
-        yield ("WPT Lint Tool", "", "lint error(s) in Web Platform Tests: exit status {0}".format(returncode))
+    if os.path.isdir(wpt_working_dir):
```

Comment:
I just want to make sure the path exists and is a directory :P
Do I have to modify it to `exists()`?